### PR TITLE
resources, media: add Figma Guide

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Add Figma Guick Start Guide
### Which version does your change affect? 

N/A

### CODE REVIEW

File: resources/figma-quick-start-guide.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+> **Note:**
+>
+> If you have a Google account, you can click **Sign up with Google** to log in with the existing Google account.
+
+![Sign up](/media/sign-up.png)
+
+### Step 2: Open the tidb-sketch-book file
+
+Currently, anyone with the link [https://www.figma.com/file/MOBwqkBtuA03agMjeGEGUT/tidb-sketch-book](https://www.figma.com/file/MOBwqkBtuA03agMjeGEGUT/tidb-sketch-book) can view [tidb-sketch-book](https://www.figma.com/file/MOBwqkBtuA03agMjeGEGUT/tidb-sketch-book).
```
Comment (lilin90) [inline resources/figma-quick-start-guide.md]:
```suggestion
Click [tidb-sketch-book](https://www.figma.com/file/MOBwqkBtuA03agMjeGEGUT/tidb-sketch-book) to view the template.
```

File: resources/figma-quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+> **Note:**
+>
+> - **tidb-sketch-book** is a Figma file that collects common shapes for figure designs. When creating a figure, you are recommended to use the objects and colors in this file to maintain a consistent style for figures.
+> - If you can't open this file, contact [Jingyi Chen](mailto:chenjingyi@pingcap.com) for help.
```
Comment (lilin90) [inline resources/figma-quick-start-guide.md]:
```suggestion
> - If you cannot open this file, contact [Jingyi Chen](mailto:chenjingyi@pingcap.com) for help.
```

File: resources/figma-quick-start-guide.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+> * To ensure a consistent style for figures, design a figure **based on shapes and the color scheme in the tidb-sketch-book file**.
+> * Set text fonts to **Ubuntu**.
+
+#### Pick a color
+
+1. Select an object and click the **Fill** value in the properties panel to open the color picker.
+
+    ![Fill](/media/fill.png)
+
+2. Click the eyedropper tool icon (keyboard shortcuts: **Ctrl**+**C** or **I**) next to the slider to open the eyedropper tool.
```
Comment (lilin90) [inline resources/figma-quick-start-guide.md]:
```suggestion
2. Click the eyedropper tool icon (keyboard shortcuts: <kbd>Ctrl</kbd>+<kbd>C</kbd> or <kbd>I</kbd>) next to the slider to open the eyedropper tool.
```

Comment (lilin90) [reply resources/figma-quick-start-guide.md]: Please use the same format for all other occurrences of keys in this PR.

File: resources/figma-quick-start-guide.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+2. Set the figure format. You can export to **PNG (recommended)**, JPG, SVG, or PDF.
+
+    ![Figure format](/media/figure-format.png)
+
+3. Click the **Export Frame X** button below **PNG** to export your figure.
+
+    ![Export Frame X](/media/export-frame-x.png)
+
+4. Use a **descriptive** name for the figure. You can use lowercase characters, numbers, and hyphens (-) in the name. **Avoid using uppercase characters, spaces, and underscores**.
```
Comment (lilin90) [inline resources/figma-quick-start-guide.md]:
```suggestion
4. Use a **descriptive** name for the figure. You can use lowercase characters, numbers, and hyphens `-` in the name. **Avoid using uppercase characters, spaces, and underscores**.
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Pull request merged**